package.json

```
@@ -9,19 +9,23 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.15.4",
+    "material-icons": "^1.13.12",
+    "next": "14.0.4",
     "react": "^18",
-    "react-dom": "^18",
-    "next": "14.0.4"
+    "react-dom": "^18"
   },
   "devDependencies": {
-    "typescript": "^5",
     "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
     "autoprefixer": "^10.0.1",
+    "eslint": "^8",
+    "eslint-config-next": "14.0.4",
     "postcss": "^8",
     "tailwindcss": "^3.3.0",
```

Mesma coisa, deixe-as fixa

```suggestion
    "@types/node": "20",
    "@types/react": "18",
    "@types/react-dom": "18",
    "autoprefixer": "10.0.1",
    "eslint": "8",
    "eslint-config-next": "14.0.4",
    "postcss": "8",
    "tailwindcss": "3.3.0",
```